### ISSUE TITLE
Dependency from aligned group included in initscript classpath and project compile configuration is not aligned

### ISSUE THREAD

**Issue by chali** [nebula-plugins/gradle-nebula-integration#54]
When there is an init script which configures init script classpath with a library which should be aligned it will cause misalignment of that group later when project dependencies are resolved.

`some-*` modules should be aligned
`some-decrypt` is used in `nebula.gradle` init script. When init script is used. The project dependencies are missaligned.

`./gradlew -I nebula.gradle dependencies --configuration compile` with a result:
```
compile - Dependencies for source set 'main' (deprecated, use 'implementation' instead).
+--- netflix.bom:runtime-platform-recommendations:4.7.15
|    \--- netflix:some-ipc-common:1.187.0
\--- netflix:common-security:latest.release -> 1.23
     \--- netflix:some-decrypt:1.140.0
          +--- netflix:some-common:1.140.0
          \--- netflix:some-ipc-common:1.140.0 -> 1.187.0
```

`./gradlew dependencies --configuration compile` has expected result:
```
compile - Dependencies for source set 'main' (deprecated, use 'implementation' instead).
+--- netflix.bom:runtime-platform-recommendations:4.7.15
|    \--- netflix:some-ipc-common:1.187.0
\--- netflix:common-security:latest.release -> 1.23
     \--- netflix:some-decrypt:1.140.0 -> 1.187.0
          +--- netflix:some-common:1.187.0
          \--- netflix:some-ipc-common:1.187.0
```

The full example project is https://github.com/nebula-plugins/gradle-nebula-integration/tree/master/initscript-classpath-breaks-alignment

**Comment by ljacomet** [comment]
Any chance you could test with a 5.1 nightly? If this passes, then the cause is indeed #18.

**Comment by chali** [comment]
Tested with the latest nightly and it worked.

**Comment by ljacomet** [comment]
Note that this means the fix for this is linked to Gradle 5.1
